pkg/asset/cluster/tfvars.go

```diff
@@ -270,6 +272,26 @@ func (t *TerraformVariables) Generate(parents asset.Parents) error {
 		if err != nil {
 			return err
 		}
+
+		baseImage := string(*rhcosImage)
+		var baseImageURL string
+
+		// If baseImage is empty then it was not provided by the user and we need to create it.
+		// For doing this we get the image URL and give it to Terraform.
+		// Image name in this case will be: <InfraID>-rhcos, i.e. user-rdd6e-rhcos
+		if baseImage == "" {
+			var err error
+			ctx, cancel := context.WithTimeout(context.TODO(), 30*time.Second)
+			defer cancel()
+
+			baseImageURL, err = rhcosplatforms.OpenStack(ctx)
```

Comment:
why are we not calling this function in the rhcos image asset https://github.com/openshift/installer/pull/2473/commits/8a4153b1f3ec097b3b560894219fa276cf63251c#diff-a5ccce89cb15a18ce4324bf3aa273a6fL81

Reply:
rhcosplatforms.OpenStack(ctx) returns a url ( https://releases-art-rhcos.svc.ci.openshift.org/art/storage/releases/rhcos-4.3/43.80.20191002.1/x86_64/rhcos-43.80.20191002.1-openstack.x86_64.qcow2), but we expect osImage function to return a Glance image name (i.e. "rhcos"), so we can't use the function there directly.